src/grt/grt-errors.adb

```diff
@@ -254,6 +254,20 @@ package body Grt.Errors is
       Fatal_Error;
    end Error;
 
+   procedure Error (Str : String;
+                    Filename : Ghdl_C_String;
+                    Line : Ghdl_I32) is
+   begin
+      Error_H;
+      Put_Err (Str);
+      Put_Err (" at ");
+      Put_Err (Filename);
+      Put_Err (" line ");
+      Put_I32 (Error_Stream, Line);
+      Newline_Err;
```

Comment:
grt.lib.ghdl_program_error only uses Error_* subprograms, and I think this is correct.
Might be worth creating a subprogram for filename + line.

Reply:
OK thanks, please ignore for now (I need to figure out rebase!)

Reply:
Tried rebasing, we'll see...
I thought exporting an Error subprogram that understood location could be useful for other runtime errors ... (note the `Put_Err` calls in the snippet above are only used in grt_errors). And some other subprograms in grt.lib use Error not Error_*, ... or I may be misunderstanding your point?